pkg/auth/cache/metrics_accesscache.go

```diff
@@ -73,6 +79,12 @@ func NewAccessCacheMetrics() AccessCacheMetrics {
 			"status",
 			"error",
 		}),
+		subjectCachedNamespaceCounter: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: "namespace_lister",
+			Subsystem: "accesscache",
+			Name:      "cached_namespace_count",
+			Help:      "number of cached namespaces",
+		}, []string{"apiGroup", "user"}),
```

Comment:
```suggestion
		}, []string{"apiGroup", "subject"}),
```

Wouldn't it be more appropriate to have subject instead of user here?